fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -356,4 +323,97 @@ public Integer getTimeout() {
     public DruidServiceConfig getServiceConfig() {
         return serviceConfig;
     }
+
+    protected Meter getHttpErrorMeter() {
+        return httpErrorMeter;
+    }
+
+    protected Meter getExceptionMeter() {
+        return exceptionMeter;
+    }
+
+    protected DruidServiceConfig getDruidServiceConfig() {
+        return serviceConfig;
+    }
+
+    /**
+     * Logs request using RequestLog and logs Druid response.
```

Comment:
This also restores the serialized request log info back into the RequestLog object, stops / starts some timers, counts outstanding queries, etc. We should definitely call those things out, since those are _very_ important side-effects of this method.